Cannot find module @feathersjs/generators/lib/connection/templates/knex.tpl

### Steps to reproduce

After running `npm create feathers@latest feathers-chat`,  it setup partially the base project, but it raises an error:

```
Error: Cannot find module '/Users/patamimbre/.npm/_npx/7433acef8618976b/node_modules/@feathersjs/generators/lib/connection/templates/knex.tpl' imported from /Users/patamimbre/.npm/_npx/7433acef8618976b/node_modules/@featherscloud/pinion/lib/utils.js
Did you mean to import /Users/patamimbre/.npm/_npx/7433acef8618976b/node_modules/@feathersjs/generators/lib/connection/templates/knex.tpl.js?
```

![CleanShot 2024-01-22 at 21 51 53@2x](https://github.com/feathersjs/feathers/assets/9404632/39c5f573-41f2-4ca2-b857-32d92cdc7c50)


### Expected behavior

Do not display the weird error

### Actual behavior

Create the project without errors

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):
create-feathers@5.0.17

--- a/packages/generators/src/connection/index.ts
+++ b/packages/generators/src/connection/index.ts
@@ -90,7 +90,7 @@ export const generate = (ctx: ConnectionGeneratorArguments) =>
         runGenerator<ConnectionGeneratorContext>(
           __dirname,
           'templates',
-          ({ database }) => `${getDatabaseAdapter(database)}.tpl`
+          ({ database }) => `${getDatabaseAdapter(database)}.tpl.js`
         ),
         mergeJSON<ConnectionGeneratorContext>(
           ({ connectionString, database }) =>

--- a/packages/generators/src/service/index.ts
+++ b/packages/generators/src/service/index.ts
@@ -210,4 +210,4 @@ export const generate = (ctx: ServiceGeneratorArguments) =>
       } as ServiceGeneratorContext
     })
     .then(runGenerators<ServiceGeneratorContext>(__dirname, 'templates'))
-    .then(runGenerator<ServiceGeneratorContext>(__dirname, 'type', ({ type }) => `${type}.tpl`))
+    .then(runGenerator<ServiceGeneratorContext>(__dirname, 'type', ({ type }) => `${type}.tpl.js`))

